Misc/NEWS.d/next/macOS/2018-05-16-13-25-58.bpo-13631.UIjDyY.rst

```diff
@@ -0,0 +1,9 @@
+The .editrc file in user's home directory is now processed correctly during
+the readline initialization through editline emulation on macOS. For
+example, let's assume the .editrc file with the following content::
```

Comment:
This is a bit too verbose for a NEWS entry.  The first sentence is sufficient; suggest removing from "For example" on.